requirements/docker.txt

```diff
@@ -86,7 +86,7 @@ cron-descriptor==1.4.5
     # via
     #   -r requirements/pip.txt
     #   django-celery-beat
-cryptography==44.0.3
+cryptography==45.0.2
```

Comment:
Nothing breaking it seems https://github.com/pyca/cryptography/blob/HEAD/CHANGELOG.rst#4502---2025-05-17